File: Ejercicios_etapa_1/Lua/Ejercicio_2/prueba.test.lua
-- Aseguramos que la carpeta actual esté en el path de búsqueda de módulos
package.path = "./?.lua;" .. package.path

local nombreModule = require("cualEsTuNombre")
local cualEsTuNombre = nombreModule.cualEsTuNombre

describe("cualEsTuNombre", function()
    it("Debe devolver 'Hola Juan Pérez!'", function()
        assert.are.equal("Hola Juan Pérez!", cualEsTuNombre("Juan", "Pérez"))
    end)

    it("Debe devolver 'Hola Ana Gómez!'", function()
        assert.are.equal("Hola Ana Gómez!", cualEsTuNombre("Ana", "Gómez"))
    end)
end)

File: Ejercicios_etapa_1/00_Javascript/Ejercicio_2/prueba.test.js
import { describe, it, expect } from "vitest"
import { cualEsTuNombre } from "./index.js" // Ensure correct path

describe("cualEsTuNombre", () => {
  it("Debe devolver 'Hola Juan Pérez!'", () => {
    expect(cualEsTuNombre("Juan", "Pérez")).toBe("Hola Juan Pérez!")
  })

  it("Debe devolver 'Hola Ana Gómez!'", () => {
    expect(cualEsTuNombre("Ana", "Gómez")).toBe("Hola Ana Gómez!")
  })
})
